integrations/extensions/starter-kits/servicenow/openapi.sn.json

```
@@ -0,0 +1,684 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "ServiceNow Blog",
+    "version": "1.0.0",
+    "contact": {}
+  },
+  "servers": [
+    {
+      "url": "https://{instance}.service-now.com",
+      "variables": {
+        "instance": {
+          "default": "dev93773",
+          "description": "Your PDI name"
+        }
+      }
+    }
+  ],
+  "components": {
+    "securitySchemes": {
+      "basicAuth": {
+        "type": "http",
+        "scheme": "basic"
+      }
+    }
+  },
+  "security": [
+    {
+      "basicAuth": []
+    }
+  ],
+  "paths": {
+    "/api/now/table/incident": {
+      "post": {
+        "summary": "https://dev93773.service-now.com/api/now/table/incident",
+        "description": "https://dev93773.service-now.com/api/now/table/incident",
```

We should change the summary, since this is part of the operation name that shows up in tooling even after the instance variable is changed, and also to be more descriptive :)
```suggestion
        "summary": "Create Incident",
        "description": "Create an incident with customer provided info.",
```

---

I deliberately left that since this is how the convert tool works

---

Hmm. I still think this should be updated because not everyone will be following the blog, and might wonder why their summary/description isnt updating in their endpoints in tooling (even though they *are* updated via the `instance` variable). Or maybe _at least_ add a comment in the readme that's like a disclaimer? Something like

"The name of the API call will show as `dev93773` for sake of consistency with the example shown in the blog post." To update this, you can modify the `summary` and `description` in the OpenAPI spec with however you'd like to name and describe the endpoint."

---

ok, did your original request, because you are right that there are good chances most people using this won't read the blog